packages/mongodb-chatbot-ui/src/useChatbot.tsx

```diff
@@ -26,6 +27,7 @@ export type ChatbotData = {
   isExperimental: boolean;
   maxInputCharacters?: number;
   maxCommentCharacters?: number;
+  initialOpen?: boolean;
```

Comment:
@branberry What's this for?

Reply:
Ah, this can be removed. I was toying around with different approaches and left this in on accident